### PR TITLE
Added a simple example for Appwrite's Locale API. 

### PR DESCRIPTION
**Pull request by k-guy-dev**
This completes the "Creating a Locale API Vue example" on issue #546 (https://github.com/appwrite/appwrite/issues/546)

### CODE REVIEW

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
This example is to showcase [Appwrite's Locale API](https://appwrite.io/docs/client/locale) with [Vue](https://vuejs.org/) by creating a simple web-app that displays your IP, Country, Continent, CountryCode, Continent and Currency.
```

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]: Add some more punctuation in this line please

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
While we are in the CLI, we will also install the Appwrite Web SDK by running:
```

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]: This line should start with an uppercase letter
```suggestion
And finally we will launch the React development server with:
```

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]: These sentences need to be looked over again.

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
TL:DR: Create a Appwrite SDK Instance and initalise it with the endpoint and ProjectID of the project we are working with. Then export this for usage outside of the file.
```

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]: That last past of the sentence is somewhat weird in my opinion.

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
Congratulations!!! We just used the Locale API to get the clients IP and other details!! Woo-hoo! 🥳🥳🥳
```

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
Good luck! If you need any help, feel free to join the [Discord](https://appwrite.io/discord) or refer to the [Appwrite Documentation](https://appwrite.io/docs).
```

**Review by stijnvdkolk — CHANGES_REQUESTED** [review]

File: example-locale_api/README.md
Comment (k-guy-dev) [reply example-locale_api/README.md]: @Kolkies , I've changed that part, can you please review it again. Thanks

File: example-locale_api/README.md
Comment (k-guy-dev) [reply example-locale_api/README.md]: Done.

File: example-locale_api/README.md
Comment (k-guy-dev) [reply example-locale_api/README.md]: Made some changes, can you review it ?

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
We'll be using the Vue CLI, to create a boilerplate and quickly get started with our project.
```

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
To initialize the Appwrite's SDK, create a `util.js` file in the `src` folder and paste the following code. Here, inside the `setEndpoint` and `setProject` parantheses, add your API endpoint and project ID from Appwrite's dashboard.
```

**Review by stijnvdkolk — CHANGES_REQUESTED** [review]

**Comment by k-guy-dev** [comment]
@Kolkies really sorry for leaving small mistakes. Committed the changes.

**Comment by stijnvdkolk** [comment]
It's ok :D

File: example-locale_api/README.md
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
To initialize the Appwrite's SDK, create a `utils.js` file in the `src` folder and paste the following code. Here, inside the `setEndpoint` and `setProject` parantheses, add your API endpoint and project ID from Appwrite's dashboard.
```

**Review by stijnvdkolk — CHANGES_REQUESTED** [review]
Should be the last change

**Review by stijnvdkolk — APPROVED** [review]

**Comment by k-guy-dev** [comment]
Committed the last change as well. I think it can be merged now.

**Comment by stijnvdkolk** [comment]
I can't merge it, @eldadfux can you do that?

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

@k-guy-dev, please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**
